### DISCUSSION TITLE
Amazon Vegetation & Climate

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/cesm_dev#26] [2. Model bias]
In B cases we've noticed **low leaf area index over the Amazon** (compared to land only CRU-JRA runs, below). 
- [low precipitation](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_PREC.gif)
- [low humidity](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_QBOT.gif), 
- [higher temperatures](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_TSA.gif), and 
- [high incoming SW](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_FSDS.gif) radiation over the region.
<p align="center"> <img src="https://github.com/user-attachments/assets/6d7bfd95-71f6-4c30-ad06-1ef4aabd32a0" width="450" /></p>

We do have some suggestions from [recent work](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2024GL108372) by @czarakas and friends suggests that land model parameters can influence regional precipitation.  How much can land model parameters help bring back rains (and clouds) over the Amazon?

#### Links above are compared to land only (I-cases), but we see similar differences between B and F-cases.

--------------
Summary of relevant LMWG diagnostics
**Relevant Land Diagnostics**
[B-120_CLM5.2 parameters vs. B-120 (CLM5.3 parameters)](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_dif_paramfile/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_dif_paramfile.11-20-b.e30_beta04.BLT1850.ne30_t232_wgx3.120_11_20/setsIndex.html)

 [B-116 vs. I-5.3](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/setsIndex.html)

[B-116 vs. B-112](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20/setsIndex.html)

[B-112 vs. B-111](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/setsIndex.html)

[B-116 vs. F-0008](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_14_23-f.e30_alpha04a.FMTHIST.ne30_L93_tuning.008_2000_2009/setsIndex.html), the F-case is a CLM6-SP case

Lots of [LMWG diagnostics posted](https://webext.cgd.ucar.edu/BLT1850/) from B cases here (thanks @olyson)

### Notes on the dry amazon bias
- Amazon biases were investigated with F-cases, see [#76](https://github.com/NCAR/LMWG_dev/issues/76) and [#77 on LMWG_dev](https://github.com/NCAR/LMWG_dev/issues/77). 
- Initial conditions were investigated with #25. 

### Notes on low LAI biases:
We identified this came in with #15, which also included updates to CLM5.3 which included a new
- **Fire Model**: investigated with [#82 in LMWG_dev](https://github.com/NCAR/LMWG_dev/issues/82)
- **Land parameterization**: investigated with [#85 in LMWG_dev](https://github.com/NCAR/LMWG_dev/issues/85)
- **Surface Data**, not assumed to be responsible for this issue 

- Additional discussion can also be found on the [LMWG_dev page](https://github.com/NCAR/LMWG_dev/discussions/75).

**Comment by wwieder** [comment]
Potentially useful here are the regional climatology here for the 
#19 (red) and an F_beta04_clm6SP (blue) case

<p align="center"> <img src="https://github.com/user-attachments/assets/7b498e84-3dd5-4af4-b15e-55cfae486c69" width="450" /></p>

**Comment by wwieder** [comment]
#### Can land parameters increase ET and precip over the Amazon? 
**Investigated with [LMWG_dev_#76](https://github.com/NCAR/LMWG_dev/issues/76) and [77](https://github.com/NCAR/LMWG_dev/issues/77).**
- As with the figure above, we are unable to replicate behavior from the coupled model in F-cases [#77 vs. #76 diags](https://webext.cgd.ucar.edu/F2000climo/f.e30_alpha04a.FMT2000BgcCrop.52params.lmwg_dev_77/lnd/f.e30_alpha04a.FMT2000BgcCrop.52params.lmwg_dev_77.11-20-f.e30_alpha04a.FMT2000BgcCrop.CTRL.lmwg_dev_76.11-20/setsIndex.html)
  - Increasing tropic LAI in F-cases also does not increase precip over the Amazon

#### Can better land initial conditions improve the situation?
**Investigated with #25**
- Initializing B cases updated initial conditions does not solve the issue, LAI declines sharply [B-120 vs. B-118 diags](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.120/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_7_16-b.e30_beta04.BLT1850.ne30_t232_wgx3.118_11_20/setsIndex.html)

#### When did we start seeing this behavior?
- LAI declines in the Amazon came in with #15, see [B-112 vs. B-104 diags](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.104_11_20/setsIndex.html)
  - Notably we see declines in LAI but without decreases in precip.
- Diffs between [B-112 vs. B-111](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/setsIndex.html)

The focus of #15 was on new ocean model tunings, but I think it is the first coupled model run that used CTSM5.3.  This introduced new datasets, parameter file, and a new fire model. Changes are pretty ubiquitous, but **reductions in tropical and boreal forest LAI** seem consistent with aims for the PPE informed calibration of land-only runs introduced in CTSM5.3. 

<p align="center"> <img src="https://github.com/user-attachments/assets/14c26670-c75c-48b3-8def-eacb8a8d7c0a" width="450" /></p>

#### Notably these decreases in Amazonian LAI are not associated with decreases in precipitation and only modest changes in air temperature

<p align="center"> <img src="https://github.com/user-attachments/assets/03d2fb4f-c688-4409-a87b-266172441649" width="450" /></p>


#### [Fire C losses do increase](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/set6/set6_frFlx_Amazonia.gif), but it seems unlikely fire alone is causing the drop in LAI?

#### There's some indication that [QBOT decreases](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/set2/set2_ANN_QBOT.gif) over southern and eastern Amazon, were our our [regional plots](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/set3/set3_reg_Amazonia.gif) focus?

**Reply by wwieder** [reply]
Some of this is review from the LMWG_dev discussion, but I'm confused by the 
1) introduction of CTSM5.3 coinciding with this change in Amazon LAI and our 
2) inability to replicate the behavior in F-cases?

@adamrher, @dlawrenncar, @linniahawkins, @djk2120, @katiedagon, @aswann, @czarakas, @olyson, @slevis-lmwg

**Comment by dlawrenncar** [comment]
I agree that the fact that this behavior is not replicated in an F case is
odd. But, it likely isn’t coincidence that this is happening with first use
of ctsm5.3.  Not sure whether it is unlikely or not that fire changes could
do this. A test would be to run 112 but reverting back to whatever CTSM was
being used in 104 or whichever is the most appropriate run. If the Amazon
problem goes away, then that is a big clue and we will have to try to
understand what looks like pretty thresholdy behavior.

**Reply by slevis-lmwg** [reply]
@wwieder @dlawrenncar I think it may be relatively simple for me to replicate the B112 case with the old fire scheme. Seems worth trying. I am more suspicious of the fire update than of the change to 5.3 datasets. But we can follow up with that right after if needed.

**Reply by slevis-lmwg** [reply]
I started what I'm calling b112oldfire: https://github.com/NCAR/LMWG_dev/issues/82
I posted there for reference all the lnd_in diffs between b112 and b111.

**Reply by wwieder** [reply]
Note, reverting to CLM5 fire does not save the [amazon](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire.11-20-b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20/set6/set6_turbf_Amazonia.gif), although  [LAI in the E. Amazon](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire.11-20-b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20/set2/set2_ANN_TLAI.gif) does beneficially increase with the old fire scheme.  See also this [regional climatology](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112oldfire.11-20-b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20/set3/set3_landf_Amazonia.gif)

**Comment by dlawrenncar** [comment]
@slevis-lmwg Thanks.  That's a lot of diffs between b112 and b111, but it does seem like fire is the most likely.  I see that there are new params for P thresholds for fire in tropical forests (?).  That looks like something that could be a factor and which could explain a somewhat thresholdy behavior between F cases and B cases.  Perhaps there is just a smallish reduction in P in B cases and then that leads to some vegetation die off and that leads to less P and there is a feedback.  If you look at F cases and B cases for P in Amazonia, it's quite a bit drier in B cases (before and after fire model changes), so I think it makes sense that we are seeing a response in B cases, but not F cases.  This really points to the new fire model as being the culprit, but we will have confirmation shortly.
 
![image](https://github.com/user-attachments/assets/73218237-70fd-4e01-8b10-67437f12ed51)
![image](https://github.com/user-attachments/assets/6e6e663e-26db-4db7-87da-02954716a614)

**Comment by wwieder** [comment]
One more note, the regional temperature and precip biases we're seeing in B-cases with CESM3 are worse that what we had in CESM1 and CESM2, [land diagnostics here](https://webext.cgd.ucar.edu/BHIST/b.e21.BHIST.f09_g17.CMIP6-historical.001/lnd/b.e21.BHIST.f09_g17.CMIP6-historical.001.1986_2005-b.e11.B20TRC5CNBDRD.f09_g16.001.1986_2005/setsIndex.html) 
![image](https://github.com/user-attachments/assets/0cc19bea-fa35-473a-a477-55d9ff6c86bb)
